app/src/main/java/com/example/firstapp/SharedPreferenceManager.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.firstapp
+
+import android.content.Context
+
+class SharedPreferenceManager(
+    private val context: Context
+) : PreferenceManager {
```

Comment:
cool, also it would be cool to read about Dependency Injection but it could be difficult on this level